srcpkgs/swayfx/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'swayfx'
+pkgname=swayfx
+version=0.2
+revision=1
+build_style=meson
+configure_args="-Dwerror=false -Db_ndebug=false"
+conf_files="/etc/sway/config"
+hostmakedepends="pkg-config wayland-devel scdoc"
+makedepends="wlroots-devel pcre2-devel json-c-devel pango-devel cairo-devel
+ gdk-pixbuf-devel libevdev-devel"
+depends="libcap-progs swaybg xorg-server-xwayland libxkbcommon>=1.5.0_1"
```

Comment:
Shouldn't be necessary anymore:
```suggestion
depends="libcap-progs swaybg xorg-server-xwayland"
```

Reply:
Thanks for the feedback, I applied that change.